src/style_manager/view/PropertyView.js

```diff
@@ -427,6 +439,37 @@ module.exports = Backbone.View.extend({
           (stylableReq.indexOf(id) >= 0 || stylableReq.indexOf(property) >= 0));
     }
 
+    // Check if the property is available based on other property's values
+    if (sectors && requires) {
+      const properties = Object.keys(requires);
+      sectors.each(sector => {
+        sector.get('properties').each(model => {
+          if (properties.includes(model.id)) {
+            const values = requires[model.id];
+            stylable = stylable && values.includes(model.get('value'));
```

Comment:
`Array.includes` is not available in IE, so use `import { includes } from 'underscore'` instead